bundle/src/test/resources/com/adobe/acs/commons/rewriter/impl/ContentVariableTransformer.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
nit: Should this be named `ContentVariableTransformerContent.json` (ending in `Content`) similar to the other test data files?

Reply:
updated